src/SimpleCalculator/UserInputs.cs

```diff
@@ -1,17 +1,23 @@
 namespace SimpleCalculator;
 
 public class UserInputs
-
 {
-    public static int GetUserInput(string message)
+    private string userInput;
+    public string UserInput
+    {
+        get { return userInput; }
+        set { userInput = value; }
```

Comment:
Another encapsulation principle is hiding functionality as much as you can if it lets a program works. In this particular case setter is not required and it's not used. Please remove it.